browser/brave_ads/android/java/org/chromium/chrome/browser/dialogs/BraveAdsSignupDialog.java

```diff
@@ -46,7 +47,7 @@ public static boolean shouldShowNewUserDialog(Context context) {
           shouldShowOnboardingDialog()
           && PackageUtils.isFirstInstall(context)
           && !BraveAdsNativeHelper.nativeIsBraveAdsEnabled(Profile.getLastUsedProfile())
-          && !BraveRewardsPanelPopup.isBraveRewardsEnabled()
+          && !BravePrefServiceBridge.getInstance().isBraveRewardsEnabled()
```

Comment:
Shouldn't `BraveRewardsPanelPopup.isBraveRewardsEnabled` be removed if we're no longer using it? Seems like it could be confusing to people if there are two methods in two different places for this